src/Components/Samples/BlazorUnitedApp/Pages/Index.razor

```diff
@@ -18,6 +18,8 @@
     <p>Submited.</p>
 }
 
+<Counter />
+
```

Comment:
This is just for manually validating this all works. I can remove it before merging the PR.